contracts/src/LightClient.sol

```diff
@@ -207,17 +234,15 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
 
         blocksPerEpoch = numBlockPerEpoch;
 
+        maxStateHistoryDuration = 86400; // set to one day epoch by default
```

Comment:
```suggestion
        maxStateHistoryDuration = 10 days;
```

Or is there a particular reason for using 1 day and not time units?

Reply:
Also shouldn't we make it possible to pass this to the `initialize` so we don't have to run a second transaction to modify it?

Reply:
yes, will do

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/ce7e74da95360a0f5abb30248549e6bf329da445